bibliopixel/drivers/WS2812SPI.py

```diff
@@ -0,0 +1,50 @@
+from . spi_driver_base import DriverSPIBase, ChannelOrder
+from .. import gamma as _gamma
+
+
+class DriverWS2812SPI(DriverSPIBase):
+    """SPI driver for WS2812(b) based LED strips on devices like the Raspberry Pi, OrangePi, BeagleBone,.."""
```

Comment:
We are seriously [`flake8`](http://flake8.pycqa.org/en/latest/) (a brutal style corrector) these days.  I think this line is too long for that...